<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bank</class>
 <widget class="QWidget" name="Bank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'맑은 고딕'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;한화은행&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="selectLoginService">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="w_login">
    <widget class="QStackedWidget" name="selectLoginSignup">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>471</width>
       <height>291</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="login">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>아이디</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>사용자암호</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>로그인</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>230</y>
         <width>88</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>회원가입</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>로그인</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="signup">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Signup</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>회원가입</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>사용자암호</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>아이디</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>회원가입</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>w_login</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="w_service">
    <widget class="QTabWidget" name="selectService">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>501</width>
       <height>331</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="w_checkAccount">
      <attribute name="title">
       <string>조회</string>
      </attribute>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>계좌조회</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_checkAccount">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>250</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌조회</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="info_checkAccount">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>441</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_makeAccount">
      <attribute name="title">
       <string>개설</string>
      </attribute>
      <widget class="QLineEdit" name="input_makeAccountPw">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>초기 입금액</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>계좌개설</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_initMoney">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_makeAccount">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌개설</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌 비밀번호</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="info_makeAccount">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>441</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_deposit">
      <attribute name="title">
       <string>입금</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[입금] 계좌목록</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_depositCheck">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌조회</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_deposit">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>입금</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>입금받을 계좌</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_receivedAccount">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_depositMoney">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>입금하실 금액</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="info_depositAccountList">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>441</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_withdraw">
      <attribute name="title">
       <string>출금</string>
      </attribute>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>출금하실 금액</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="info_depositAccountList_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>441</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[출금] 계좌목록</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_receivedAccount_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_depositCheck_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌조회</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_deposit_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>출금</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>출금할 계좌</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_depositMoney_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>200</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌 비밀번호</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_depositMoney_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>230</y>
         <width>221</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_sendMoney">
      <attribute name="title">
       <string>이체</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>w_service</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>textBrowser</zorder>
  <zorder>selectLoginService</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
